Source/towners.h

```diff
@@ -23,11 +23,6 @@ typedef struct TownerStruct {
 	int _ttype;
 	int _tx;    // Tile X-position of NPC
 	int _ty;    // Tile Y-position of NPC
-	int _txoff; // Sprite X-offset (unused)

```

Comment:
Keep these fields for towner walking animations?

Reply:
Agreed, or just comment them out. Those fields are needed to make the towners walk. I guess in modern times everyone is too obese to move around :s

Reply:
was kinda hoping we wold have some saner movement logic first, but ok  i will keep i around until that time comes :)